pydantic/mypy.py

```diff
@@ -210,6 +215,9 @@ def __init__(self, options: Options) -> None:
                 setting = plugin_config.getboolean(CONFIGFILE_KEY, key, fallback=False)
                 setattr(self, key, setting)
 
+    def to_data(self) -> Dict[str, Any]:
```

Comment:
Do we need a separate method for this?

Reply:
Not necessarily. Just though it might be best to let the `PluginConfig` class handle it.

Alternatives would be to move the dict build directly to `PydanticPlugin.__init__` or create a separate `data` attribute for `PluginConfig`. The later one is a bit tricky as it would also require a dedicated `fields` attribute then.

Reply:
Ok, keep as is.